### DISCUSSION TITLE
docs: 发现文档中有一个多余符号

### DISCUSSION THREAD

**Discussion by ChasLui** [fireboomio/product-manual#109] [Q&A]
这个链接下有个多余的问号
https://docs.fireboom.io/er-ci-kai-fa/mo-ban-gui-fan
�
<img width="995" alt="image" src="https://github.com/fireboomio/product-manual/assets/10083758/d1559ce2-1a95-48ef-962d-3ff016ed6dd1">

**Comment by AnsonCode** [comment]
收到，感谢，我们来处理下。
